docs/artifactory/repocreate/help.go

```diff
@@ -0,0 +1,8 @@
+package repocreate
+
+const Description = "Create a new repository in Artifactory."
+
+var Usage = []string{`jfrog rt rc <template path>`}
+
+const Arguments string = `	template path
+		Specifies the local file system path for the template file to be used for the repository creation.`
```

Comment:
Let's add to the end of the description:
The template can be created using the "jfrog rt rpt <template path>" command.
Let's add this to the repoupdate help as well.